app/uploaders/logo_svg_uploader.rb

```diff
@@ -0,0 +1,47 @@
+class LogoSvgUploader < BaseUploader
+  # This Uploader is being used by a Data Update Script
+  # lib/data_update_scripts/20211208074428_migrate_logo_svg_data.rb
+  # to convert logo_svg contents to a png file.
+
+  STORE_DIRECTORY = "uploads/logos/".freeze
+  EXTENSION_ALLOWLIST = %w[svg png].freeze
+  IMAGE_TYPE_ALLOWLIST = %i[svg png].freeze
+  CONTENT_TYPE_ALLOWLIST = %w[image/svg+xml image/png].freeze
+
+  process convert: "png"
+
+  def store_dir
+    STORE_DIRECTORY
+  end
+
+  def extension_allowlist
+    EXTENSION_ALLOWLIST
+  end
+
+  def image_type_whitelist
+    # this is needed by CarrierWave::BombShelter
+    IMAGE_TYPE_ALLOWLIST
+  end
+
+  def content_type_allowlist
+    CONTENT_TYPE_ALLOWLIST
+  end
+
+  def filename
+    # random_string in the filename to avoid caching issues
+    "original_logo_#{random_string}.png"
+  end
+
+  version :resized_logo do
+    process resize_to_limit: [nil, 80]
+    def full_filename(_for_file = file)
+      "resized_logo_#{random_string}.png"
+    end
+  end
+
+  private
+
+  def random_string
+    SecureRandom.alphanumeric(20)
```

Comment:
(non-blocking) Do we want to memoize this?  Wondering if we want `original_logo_asdfasdf.png` and `resized_logo_asdfasdf.png`?

Reply:
I mean I don't think it would hurt, but maybe not a priority since this most likely a once-off script that we're running for existing Forems to migrate their existing logos. 

I could go in and optimise this on LogoUploader as that is used for our normal upload process, I'll go in and do this in a subsequent PR.